lib/system.nim

```diff
@@ -551,6 +551,10 @@ when notJSnotNims and not defined(nimSeqsV2):
 when notJSnotNims:
   include "system/hti"
 
+  proc writeToStdErr(msg: cstring, length: int)
+  proc writeToStdErr(msg: cstring)
+  proc writeToStdErr(msg: string)
```

Comment:
ditto, this could be defined directly as:
```nim
  proc writeToStdErr(msg: string) {.inline.} =
    writeToStdErr(msg.cstring, msg.len)
```

that way, the only proc to define on each platform is `proc writeToStdErr(msg: cstring, length: int)`